Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -3512,8 +3512,18 @@ private static View getInvoiceLayout(JSONObject data, JSONObject selectedRide, J
         textView.setText(fares.getString("pickupCharges"));
         textView = invoiceLayout.findViewById(R.id.nominalFare);
         textView.setText(fares.getString("nominalFare"));
+        if(fares.getString("nominalFare").equals("₹ 0.0")){
```

Comment:
changing visibility is not a solution we may get more break ups from backend change this accordingly. and use `fareList` instead `fareBreakUpList`